spec/graphql/queries/practices/therapists/get_therapists_spec.rb

```diff
@@ -0,0 +1,155 @@
+    @therapist_5 = Therapist.create!(address: Faker::Address.full_address, phone_number: Faker::PhoneNumber.cell_phone, image_url: Faker::LoremFlickr.image, bio: "even more different", labels: ["Works with Depression", "Works with Anxiety"], practice_id: @practice_3.id, name: "Bob McKay")
+  end
+
+  describe 'display therapists' do
+    it 'can query therapists' do
+      result = IotnboBeSchema.execute(query).as_json
+      expect(result["data"]).to be_a Hash
+      expect(result["data"]).to have_key("therapists")
+      expect(result["data"]["therapists"]).to be_a Array
+      expect(result["data"]["therapists"].count).to eq 5
+
+      result["data"]["therapists"].each do |therapist|
+        expect(therapist).to have_key("id")
+        expect(therapist).to have_key("name")
+        expect(therapist).to have_key("phoneNumber")
+        expect(therapist).to have_key("imageUrl")
+        expect(therapist).to have_key("bio")
+        expect(therapist).to have_key("labels")
+        expect(therapist).to have_key("practice")
+        expect(therapist).to have_key("address")
+      end
+    end
+
+    it 'can query therapists with certain label' do
+      result = IotnboBeSchema.execute(query_labels).as_json
+      expect(result["data"]).to be_a Hash
+      expect(result["data"]).to have_key("therapists")
+      expect(result["data"]["therapists"]).to be_a Array
+      expect(result["data"]["therapists"].count).to eq 2
+
+      result["data"]["therapists"].each do |therapist|
+        expect(therapist).to have_key("id")
+        expect(therapist).to have_key("name")
+        expect(therapist).to have_key("phoneNumber")
+        expect(therapist).to have_key("imageUrl")
+        expect(therapist).to have_key("bio")
+        expect(therapist).to have_key("labels")
+        expect(therapist).to have_key("address")
+        expect(therapist).to have_key("practice")
+        expect(therapist["practice"]).to be_a Hash
+
+        expect(therapist["practice"]).to have_key("id")
+        expect(therapist["practice"]).to have_key("name")
+        expect(therapist["practice"]).to have_key("websiteUrl")
+      end
+    end
+
+    it 'can query therapists and return a uniq array' do
+      result = IotnboBeSchema.execute(query_multiple_labels).as_json
+      expect(result["data"]).to be_a Hash
+      expect(result["data"]).to have_key("therapists")
+      expect(result["data"]["therapists"]).to be_a Array
+      expect(result["data"]["therapists"].count).to eq 3
+
+      result["data"]["therapists"].each do |therapist|
+        expect(therapist).to have_key("id")
+        expect(therapist).to have_key("name")
+        expect(therapist).to have_key("phoneNumber")
+        expect(therapist).to have_key("imageUrl")
+        expect(therapist).to have_key("bio")
+        expect(therapist).to have_key("labels")
+        expect(therapist).to have_key("address")
+        expect(therapist).to have_key("practice")
+        expect(therapist["practice"]).to be_a Hash
```

Comment:
Brilliant test for multiple labels! Should we add one additional line to this test that expects the three labels in the query exactly match the labels returned for these three therapists?